distribution/lib/Standard/Base/0.0.0-dev/src/System/File_Format.enso

```diff
@@ -0,0 +1,46 @@
+from Standard.Base import all
+from Standard.Base.Error.Common import unimplemented
+
+type File_Format
+    ## If the File_Format supports reading from the file, return a configured instance.
+    can_read : File -> Any
```

Comment:
```suggestion
    can_read : File -> Any | Nothing
```

I know that semantically `Any | Nothing =:= Any`, but I suggest it may be worth including it explicitly here to clearly mark that `Nothing` means "cannot read".

Still, I'd update the docstring to say what is returned if it doesn't support reading - because currently it is not explicitly stated.

I'm thinking that the name `can_read` is a bit confusing if it not only checks that but also returns an instance. But honestly I'm not sure if I have a better idea, so probably let's juts keep it. Maybe `configure_format_for`? Doesn't sound too great though.